modules/installation-osp-setting-worker-affinity.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+
+[id="installation-osp-setting-worker-affinity_{context}"]
+= Setting compute machine affinity
+
+Optionally, you can set the affinity policy for compute machines during installation.
+
+[TIP]
+====
+The installer does not select an affinity policy for compute machines by default. 
+
+Control plane machines are created with a `soft-anti-affinity` policy. 
+
+You can learn more about link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/configuring_the_compute_service_for_instance_creation/configuring-instance-scheduling-and-placement[{rh-openstack} instance scheduling and placement] in the {rh-openstack} documentation.
+====
+
+You can also create machine sets that use particular {rh-openstack} server groups after installation. 
+
+.Prerequisites
+
+* Create the `install-config.yaml` file and complete any modifications to it.
+
+.Procedure
+
+. Using the {rh-openstack} command-line interface, create a server group for your compute machines. For example:
++
+[source,terminal]
+----
+$ openstack \
+    --os-compute-api-version=2.15 \ <1>
+    server group create \
+    --policy anti-affinity \
+    my-openshift-worker-group
+----
++
+[INFO]
+====
+For more information, see the link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/15/html/command_line_interface_reference/server#server_group_create[`server group create` command documentation].
+====
+
+. Change to the directory that contains the installation program and create the manifests
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
++
+where:
++
+`installation_directory` :: Specifies the name of the directory that contains the `install-config.yaml` file for your cluster.
+
+. From the newly generated manifest files, open the `MachineSet` definition file, which is in the `manifests` directory and is named `99_openshift-cluster-api_worker-machineset-0.yaml`.
```

Comment:
Maybe s/newly generated/newly-generated/

If you are open to suggestion:
* Start a new sentence after "...definition file."
* Maybe specify the path like you did on line 121.  This seems to be a description of the path to the file.
* Are these two steps actually one step?  Is the goal to edit the machine set file and add the field?